test/membrane_live_web/channels/event_channel_test.exs

```diff
@@ -12,33 +12,85 @@ defmodule MembraneLiveWeb.EventChannelTest do
         title: "Test webinar"
       })
 
-    %{uuid: uuid}
+    {:ok, _reply, pub_socket} =
+      MembraneLiveWeb.EventSocket
+      |> socket("event_id", %{})
+      |> subscribe_and_join(MembraneLiveWeb.EventChannel, "event:#{uuid}", %{
+        name: "Mark"
+      })
+
+    %{uuid: uuid, pub_socket: pub_socket}
   end
 
-  test "check if users are in presence", %{uuid: uuid} do
+  test "check if users are in presence", %{uuid: uuid, pub_socket: pub_socket} do
     socket1 = MembraneLiveWeb.EventSocket |> socket("event_id1", %{})
-    socket2 = MembraneLiveWeb.EventSocket |> socket("event_id2", %{})
 
     {:ok, _reply, socket1} =
       subscribe_and_join(socket1, MembraneLiveWeb.EventChannel, "event:#{uuid}", %{
         name: "John"
       })
 
-    {:ok, _reply, socket2} =
-      subscribe_and_join(socket2, MembraneLiveWeb.EventChannel, "event:#{uuid}", %{name: "Adam"})
-
     pres = Presence.list("event:#{uuid}")
-    assert is_map_key(pres, "John") and is_map_key(pres, "Adam")
+    assert is_map_key(pres, "John") and is_map_key(pres, "Mark")
 
     Process.unlink(socket1.channel_pid)
     close(socket1)
 
     pres = Presence.list("event:#{uuid}")
-    assert is_map_key(pres, "Adam") and not is_map_key(pres, "John")
+    assert is_map_key(pres, "Mark") and not is_map_key(pres, "John")
 
-    Process.unlink(socket2.channel_pid)
-    close(socket2)
+    Process.unlink(pub_socket.channel_pid)
+    close(pub_socket)
 
     assert Presence.list("event:#{uuid}") == %{}
   end
+
+  test "set and remove presenter", %{uuid: uuid, pub_socket: pub_socket} do
+    @endpoint.subscribe("private:#{uuid}:Mark")
+
+    push(pub_socket, "presenter_prop", %{
+      "moderator" => "private:#{uuid}:Mark",
+      "presenter" => "private:#{uuid}:Mark"
+    })
+
+    assert_broadcast("presenter_prop", %{moderator: _})
```

Comment:
I would move it to function as we use it in both tests. It could be something as `add_presenter`.